assets/xml/misc/scene_texture_03.xml

```diff
@@ -0,0 +1,5 @@
+<Root>
+    <File Name="scene_texture_03" Segment="6">
+        <Blob Name="scene_texture_03_Blob_002D40" Size="0x4800" Offset="0x0"/>
```

Comment:
Most of this seems to be trees in various formats, but beyond the first couple it's not really guessable, annoyingly.